### DISCUSSION TITLE
detection of uc mode in some times

### DISCUSSION THREAD

**Discussion by Magic4k** [seleniumbase/SeleniumBase#2742] [Q&A] [answered]
by signing up in website "https://www.kucoin.com/ucenter/signup" using phone number , hcaptcha appears and bypass successfully  but in some cases after doing this task repetitvely it doesn't pass the hcaptcha I want to know what is the problem here noted that I use sb.driver.type() , sb.driver.uc_click() methods and here is my snippet of code : 
``
with SB(uc=True) as sb:
            url = "https://www.kucoin.com/ucenter/signup"
            sb.driver.delete_all_cookies()
            sb.driver.uc_open_with_reconnect(url)
            #write phone
            sb.driver.type('#account > input',phone)
            #changing the country code by openning list 
            sb.driver.uc_click('#account > div.KuxInput-prefix.lrtcss-18ia7lk > div > div > div > span')
            #searching for neigar
            sb.driver.uc_click('#account > div.KuxInput-prefix.lrtcss-18ia7lk > div > div > div.KuxDropDown-popper.KuxDropDown- 
                                                                            open.customDropdown > div > div > div.overlayHeader > div > input')
                                                                            
            sb.driver.type('#account > div.KuxInput-prefix.lrtcss-18ia7lk > div > div > div.KuxDropDown-popper.KuxDropDown- 
                                                                           open.customDropdown > div > div > div.overlayHeader > div > input',"234")
             
             #Clicking on Neigar                                                             
            sb.driver.uc_click('#account > div.KuxInput-prefix.lrtcss-18ia7lk > div > div > div.KuxDropDown-popper.KuxDropDown- 
                                                                        open.customDropdown > div > div > div.overlayBody > div > div.KuxBox-root.lrtcss- 
                                                                           loshnn > div > span')
            
            #clicking on creating account                                                              
            sb.driver.uc_click('#root > div > div > div > div > div.lrtcss-1cagiq7.essyzlb0 > div > div > div > div.lrtcss-1gkb030.ex3bcaf27 
                                                                      > div > form > button')
            
            #Switching to iframe                                                         
            sb.switch_to_frame("/html/body/div[6]/div/div/iframe")
            #clicking check box
            sb.driver.uc_click('#anchor-tc')
``
(also noted that : if I stopped the script for a while (2:3 min) and then rerunning it , bypasses successfully)
what is the reason here , what do u recommend to do in this case ? 
**Thanks in advance for your great effort at all** ♥
![Screenshot 2024-05-02 203401](https://github.com/seleniumbase/SeleniumBase/assets/120425697/83d691db-baae-4bb1-8192-b92a8936313f)

**Comment by mdmintz** [comment, marked as answer]
You mentioned: `"after doing this task repetitively it doesn't pass"`.
That means your IP Address was blocked for unusual traffic.
When that happens, you have to switch IP addresses (via the `proxy` arg).

UC Mode only prevents websites from detecting your automation directly.
It doesn't stop websites from guessing that you're a bot if the web traffic looks unusual.
